### ISSUE TITLE
npm out of sync for 0.3.1?

### ISSUE THREAD

**Issue by iqrow** [Khan/simple-markdown#42]
Hi, first of all thanks for all the work on this project!

I'm wondering if the code on npm is out of sync with this repo and needs updating? It shows 0.3.1 in the releases here, but there is code that is not present on the 0.3.1 on npm.

Specifically the `reactElement` function

npm 0.3.1
```
    // This should just override an already present element._store, which
    // exists so that the class of this object doesn't change in V8
    element._store = {
        validated: true,
        originalProps: element.props
    };
    return element;
};
```

github 0.3.1
```
    return {
        $$typeof: TYPE_SYMBOL,
        type: type,
        key: key,
        ref: null,
        props: props,
        _owner: null
    };
```

The reason I noticed is because I'm trying to implement unit testing for my own rules, which I will create a separate ticket for.

Thanks again.

**Comment by ariabuckles** [comment]
That's not good!

Thanks for pointing this out. It looks like the npm version is out of date compared to the github version. I'll publish a 0.3.2 and hopefully get them back to parity

**Comment by ariabuckles** [comment]
Okay, here's what was going on: 0.3.1 corresponds to 3d0d2bd973463ae51b2c61d5a05fd0b570d452d3 (tagged 0.3.1 in github, or npm simple-markdown@0.3.1)

There have been a few commits since then, most notably being react 16 compat. I had been holding off on shipping those because I'm working on a v1.0 version that will include all the fixes.

But I didn't realize the latest version on npm didn't have the react 16 fixes, and it totally should, so I just published 0.3.2 which is up to date with master.

(As a sidenote: I'm working on some pretty major api work for 1.0, hopefully to ship that soon.)

**Comment by iqrow** [comment]
Oh excellent, thank you very much for your speedy response and fix!
